velox/dwio/text/RegisterTextWriter.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+namespace facebook::velox::text {
```

Comment:
nit: how about `facebook::velox::dwio::text`?

Reply:
But for orc/parquet, their namespace doesn't have dwio

https://github.com/facebookincubator/velox/blob/c04652462250469d759396aac9d68d441ac1eb2a/velox/dwio/parquet/writer/Writer.cpp#L37
https://github.com/facebookincubator/velox/blob/c04652462250469d759396aac9d68d441ac1eb2a/velox/dwio/orc/reader/OrcReader.h#L22

Reply:
It might be better to have dwio namespace there. We probably need to fix that cc @majetideepak